### ISSUE TITLE
Questions on the Kolmogorov flows code

### ISSUE THREAD

**Issue by cjunho** [neuraloperator/physics_informed#27]
I am Junho, a researcher who is studying about PINN at SungKun Korea. At my paper, I am going to write and refer your results about Kolmogorov flows implemented by Physics-Informed Neural Operator. To do that, I would like to ask you questions about your code and the numerical results. 

1.	Could you confirm if the result I reproduced is correct? I ran your code “python3 train_operator.py --config_path configs/pretrain/Re500-pretrain-05s-4C1.yaml” with data NS_fft_Re500_T4000.npy  when setting ic_loss=1, f_loss=1, xy_loss=0. As a result, I got 0.35570 data L2 error when 40000 epochs as shown in ns_err.txt. Is it correct and can I refer the error at my paper? 

2.	I am wondering what Equation error is?

3.	What is instance wise learning fine-tuning and how can I run it at your code? At your paper, “Physics-Informed Neural Operator for Learning Partial Differential Equations” you said instance wise learning fine-tuning produced more accurate results. The instance wise learning fine-tuning looks like an additional operator loss to PDE loss made from PDE residual, but I could not find where the operator loss is at your code. 

Thank you for reading my questions.
[ns_error.txt](https://github.com/neuraloperator/physics_informed/files/12264785/ns_error.txt)
[Re500-pretrain-05s-4C1.txt](https://github.com/neuraloperator/physics_informed/files/12264791/Re500-pretrain-05s-4C1.txt)

**Comment by ROY1228W** [comment]
I have the same question as yours about the code for instance-wise fine-tuning phase. Hope the contributor will answer it.
Have you already found it? If yes, please let me know.
Many thanks in advance.
